.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: objective-c
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->